src/raft/raft.go

```diff
@@ -153,7 +156,7 @@ func (r *Raft) Stop() error {
 	return nil
 }
 
-// init all peers for raft.Peers from RaftConfig.Peers
+// init all peers for raft.Peers/raft.IdlePeers from RaftConfig.Peers/RaftConfig.IdlePeers
```

Comment:
init all peers for raft, peers from RaftConfig.Peers; idlePeers from RaftConfig.IdlePeers ?